_posts/2018-07-09-PH-espanol-in-DH2018.md

```diff
@@ -0,0 +1,32 @@
+
+[Maria José Afanador](https://github.com/mariajoafana), one of our Spanish-language editors, presented a paper *[The Programming Historian en español: Strategies and challenges for building a global DH community](https://dh2018.adho.org/the-programming-historian-en-espanol-estrategias-y-retos-para-la-construccion-de-una-comunidad-global-de-hd/)* in a panel called *Local DH, non-standardization*. The presentation, which was delivered in Spanish with bilingual slides, introduced *Programming Historian* to the audience, presented the progress of Spanish-language editorial team, discussed data on the impact of the project, and reflected on the challenges of the project moving forward.
+
+The presentation discussed the achievements of the *Programming Historian en español*. Since its launch on March 2017, the Spanish-language team has translated 33 tutorials from English to Spanish with the support of a growing network of collaborators from countries includign Spain, Mexico, Colombia, Argentina, and the United States. As the number of tutorials available in Spanish increased and the team worked to make *Programming Historian* known in Latin America and Spain, the number of visitors from Spanish-speaking countries in grew dramatically. In just a year (May 2017 to May 2018), the number of new visitors from several Spanish-speaking countries grew by as much as 1000%.    
+
+The growth of the *Programming Historian en español*, has also brought about crucial discussions on the internationalization of the project. The challenges of translating and adapting *Programming Historian* tutorials into Spanish has [led the team to consider the question of how to best reach a global audience while making our tutorials accessible and meaningful in a range of cultural contexts](https://github.com/programminghistorian/jekyll/issues/651). Expanding on this, the presentation reflected on the challeges of incentivating the production of Spanish-language original tutorials in regions such as Latin America, where the digital divide often means limited access to resources for building basic technlogical infrastructures for digital humanities work. Moving forward, the *Programming Historian en español* is [holding a Programming Historian tutorial writing workshop](https://programminghistorian.org/posts/convocatoria-taller-PH-espanol) which will bring together Spanish-speaking scholars from North, Central, and South America in an effort to produce the first original Spanish-language tutorials.
+
+<p><figure><a href="https://twitter.com/visceral_blot/status/1012453500595290112">
+        <img src="images/dh2018blog/talk-grab.png" alt=""/></a><figcaption>
```

Comment:
This needs a leading slash: `/images...` instead of `images...`

Reply:
Done.